platforms/dotnet/versionsToBuild.txt

```diff
@@ -70,6 +70,8 @@
 
 6.0.100, cb0d174a79d6294c302261b645dba6a479da8f7cf6c1fe15ae6998bc09c5e0baec810822f9e0104e84b0efd51fdc0333306cb2a0a6fcdbaf515a8ad8cf1af25b,
```

Comment:
As we are removing 6.0.0, is it possible for us to remove it from the versions to build as well?

Reply:
Historically we have not removed old versions from this file. @qianz2  would you happen to know if it'll be ok to remove phased out versions from this file?

Reply:
We have another verionsToBuild_{date}.txt file to store unsupported old version, such as 'php/versionsToBuild_11172020.txt', because we won't need to build those old versions and to avoid issue unable to find old verisons. We can gradually move unsupported old versions to there. But for dotnet 6.0.0, it's comparably latest so we continue to keep it in here.

Reply:
Thanks @william-msft and @qianz2. So at this point, once this PR is checked in, could you please confirm that we would still be able to build using version 6.0.0? If that's still the case, then I agree I'm fine with keeping this version in the versionsToBuild.txt file.

Reply:
Yes, we still have dotnet 6.0.0 SDKs in storage account. Here we only updated the minor versions of dotnet 6 runtime image We're able to build dotnet 6.0.0 apps.

Reply:
OK, thanks @qianz2!